README.md

```diff
@@ -12,6 +12,11 @@ resource "aws_route53_zone" "default" {
 module "cert" {
   source = "github.com/azavea/terraform-aws-acm-certificate?ref=0.1.0"
 
+  providers = {
+    aws.acm_account     = "aws.cert-account"
+    aws.route53_account = "aws.route53-account"
```

Comment:
I think that it would be valuable to include `provider` blocks external to the module in the example here, kinda like in the 0.11 upgrade guide:

```hcl
provider "aws" {
  region = "us-east-1"
  alias  = "use1"
}

provider "aws" {
  region = "us-west-2"
  alias  = "usw2"
}

...

providers = {
  aws.acm_account     = "use1"
  aws.route53_account = "usw2"
}
```